### DISCUSSION TITLE
What are all these pooling keys (Farmer_PK, Pool_PK, Pool Contact address) and where do they go?

### DISCUSSION THREAD

**Discussion by DroptheHammer** [ericaltendorf/plotman#892] [Q&A]
I've followed the pooling FAQ and created a plotNFT.  I have (in the gui) many items that have names which do not match the plotman config file.  I've also run "chia plotnft show" and received yet another list of variables where the names dont match the GUI or the plotman config.  What gives.. which of these go where??

Plotman Config has.

1. farmer_pk
2. pool_pk
3. pool_contract_address

GUI Has

1. wallet receive address
2. farmer reward address
3. pool reward address
4. autogenerated name from pool contract address
5. launcher id
6. pool login link

Chia PlotNFT Show displays

1. Launcher ID (This is the ONLY one which matches any of the others so far...)
2. Target address
3. owner public key  (is this MINE? owner of the wallet or owner of the pool (which is not me)
4. P2 singleton address
5. current pool URL
6. payout instructions

Seriously... How could this be more poorly thought out when none of the nomenclature matches?  Someone needs to put out a guide that actually says "paste A to variable 1 and paste B to variable 2 etc.".  How do I know what plotman config wants?

**Comment by altendky** [comment]
https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-User-Guide#step-4-add-plots

"If you are using the CLI or a 3rd party plotter, hover your mouse over the question mark in the Plot NFT page, and copy the Pool Contract Address."

Also, the full label from `chia plotnft show` is `P2 singleton address (pool contract address for plotting):`.

In both cases it uses the same term as the plotman configuration option.

I understand there is a lot going on and various terminology isn't as straight forward as it could (should?) be.  Documentation for plotman is one of the many things I want to improve.  But, the insults really aren't necessary.
